BUILD/task_order/default.dat

```diff
@@ -0,0 +1,94 @@
+LX_fightTentacle
+LX_freeCombatsTask	LX_freeCombatsTask_condition
+catBurglarHeistTask
+chateauPainting
+LX_faxing
+LX_artistQuest
+LX_galaktikSubQuest
+L9_leafletQuest
+L5_findKnob
+L12_sonofaPrefix
+LX_burnDelay
+LM_edTheUndying
+LX_lowkeySummer
```

Comment:
I'd like to find a good way to factor these out of the default route, but due to them not being right at the beginning I'm not sure how best to do that...

One option would be to split the prior functions in to `default_prefix` route, the following in to a `default_suffix` route, and have the default route consist of
```
route	default_prefix
route	default_suffix
```
Adding special casing where if you have route in the first column it proceeds through the entirety of the route named in the second column. Then the Ed route would be
```
route	default_prefix
LM_edTheUndying
route	default_suffix
```
And LKS route could be
```
route	default_prefix
LX_lowkeySummer
route	default_suffix
```
But ugh then there's `LX_koeInvaderHandler` further down, so there would need to be a `default_middle` as well... At that point it might be better to just keep it this way... Although those do seem like the two main splitting points, what with one being right after generic whatever stuff and another being right before the tower and some last ditch effort stuff.

Thoughts?

Reply:
Honestly it'd be duplication but I'd just make a copy of the data file for each of those paths with the added entries, and put a comment at the top to keep them in sync if any changes are made to the default routing.